src/styles/blocks/card.scss

```diff
@@ -0,0 +1,96 @@
+.card {
+  padding: 32px 16px 16px;
+  width: 200px;
+
+  // height: 408px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  transition: transform $transition-time ease;
+
+  &__picture {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 3px 40px;
+  }
+
+  &__title {
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+    margin-bottom: 4px;
+  }
+
+  &__product {
+    &-code {
+      font-size: 10px;
+      color: $text-secondary-color;
+      line-height: 14px;
+      margin-bottom: 16px;
+    }
+
+    &-buy {
+      width: 100%;
+      height: 40px;
+      background-color: $button-color;
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      text-decoration: none;
+      color: #fff;
+      font-weight: 700;
+      text-transform: uppercase;
+      border-radius: 5px;
+      font-size: 14px;
+      line-height: 16px;
+    }
+
+    &-buy:hover {
+      color: $button-color;
+      background-color: #fff;
+      border: 1px solid $button-color;
+      transition: 0.3s ease;
+    }
+  }
+
+  &__review {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    margin-bottom: 24px;
+
+    &-count {
+      font-size: 10px;
+      line-height: 14px;
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
+
+    &-label {
+      font-size: 12px;
+      line-height: 18px;
+      color: $text-secondary-color;
+    }
+
+    &-amount {
+      font-weight: 700;
+      font-size: 16px;
+      line-height: 18px;
+    }
+  }
+
+  &:hover {
+    transform: scale(120%);
```

Comment:
The transform scale value should be a unitless number. For example, `1.2` instead of `120%`. This will make the card scale up by 20%.